static/index.html

```diff
@@ -182,6 +182,15 @@
         <div id="build">
           <a target="_blank" rel="noopener noreferrer"></a>
         </div>
+        <a href="https://github.com/ubiquity/work.ubq.fi/discussions/53" target="_blank">
+          <div class="faq-icon">
+            <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 -960 960 960" width="24px" fill="#808080">
+              <path
+                d="M478-240q21 0 35.5-14.5T528-290q0-21-14.5-35.5T478-340q-21 0-35.5 14.5T428-290q0 21 14.5 35.5T478-240Zm-36-154h74q0-33 7.5-52t42.5-52q26-26 41-49.5t15-56.5q0-56-41-86t-97-30q-57 0-92.5 30T342-618l66 26q5-18 22.5-39t53.5-21q32 0 48 17.5t16 38.5q0 20-12 37.5T506-526q-44 39-54 59t-10 73Zm38 314q-83 0-156-31.5T197-197q-54-54-85.5-127T80-480q0-83 31.5-156T197-763q54-54 127-85.5T480-880q83 0 156 31.5T763-763q54 54 85.5 127T880-480q0 83-31.5 156T763-197q-54 54-127 85.5T480-80Zm0-80q134 0 227-93t93-227q0-134-93-227t-227-93q-134 0-227 93t-93 227q0 134 93 227t227 93Zm0-320Z"
+              />
+            </svg>
+          </div>
+        </a>
```

Comment:
> You should match the styles of the #build element. Match the HTML structure as well.

https://github.com/ubiquity/pay.ubq.fi/pull/240#issuecomment-2157430347

Reply:
@0x4007 Matched the build styles. I increased the opacity slightly compared to what the build style has because the icon was almost invisible. But other than that everything else is same. I think

Reply:
As far as I understand `0x4007` meant that:
1. `div#build` and `a.faq-icon` should be placed inside the same parent html tag (which is fine now)
2. `div#build` and `a.faq-icon` should be vertically aligned on the same level (which seems to be wrong now)

<img width="500" alt="Screenshot 2024-06-25 at 11 57 54" src="https://github.com/ubiquity/pay.ubq.fi/assets/119500907/1586acec-66a6-42c1-9025-512ec59c20c7">

Reply:
![Screenshot 2024-06-25 100942](https://github.com/ubiquity/pay.ubq.fi/assets/33989423/8f6ee1e8-141c-41ac-b14a-d8aa39f805c0)

![Screenshot 2024-06-25 101225](https://github.com/ubiquity/pay.ubq.fi/assets/33989423/e83c7bb9-929b-4f7f-be1d-99b9cdabc6ea)


@rndquu here's both the desktop and mobile view of the alignment of both on my end! Not sure why I looks misaligned on your end.

Reply:
The point is that the only way to vertically align them ideally is to use css flexbox or smth similar. Hardcoded paddings/margins only make it look like they're aligned but on different screen dimensions you may see both of the elements are not on the same level vertically.

<img width="500" alt="Screenshot 2024-06-25 at 13 00 03" src="https://github.com/ubiquity/pay.ubq.fi/assets/119500907/1ab60747-381c-4fbc-b67e-ecc43240526a">

Reply:
oh I get. @rndquu  I've used flex to align. Thanks